Tests/UnitTests/Purchasing/OfferingsTests.swift

```diff
@@ -153,6 +153,59 @@ class OfferingsTests: TestCase {
         expect(offerings.current) == offerings["offering_a"]
     }
 
+    func testOfferingsWithMetadataIsCreated() throws {
+        @DefaultDecodable.EmptyDictionary
```

Comment:
Also reminder to remove this

Reply:
I think this is needed though 🤔 I couldn't get it to compile without that 😬